packages/legend-graph/src/graph-manager/protocol/pure/v1/model/packageableElements/mapping/V1_MappingInclude.ts

```diff
@@ -17,14 +17,25 @@
 import { type Hashable, hashArray } from '@finos/legend-shared';
 import { CORE_HASH_STRUCTURE } from '../../../../../../../graph/Core_HashUtils.js';
 
-export class V1_MappingInclude implements Hashable {
+export abstract class V1_MappingInclude implements Hashable {
+  abstract get identifier(): string;
```

Comment:
We do we need this `identifier` ?

Reply:
We use this to prevent duplicate imports

Reply:
We avoid writing extra logic in protocol except hash code, so let's avoid doing this, the approach is to create an extension mechanism `V1_getExtraMappingIncludeIndentifierBuilders`. Thanks!